zulipterminal/ui_tools/boxes.py

```diff
@@ -117,13 +117,14 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
 
 class MessageBox(urwid.Pile):
```

Comment:
You don't need to name the commit summary as 'unfactored'; only describe what each does. The refactoring commit can start with 'refactor <something>...', but you're describing a change in each case, rather than a 'state'.

I was expecting to see tests in this commit, to go with the extra parameter.